app/settings/tokens/-components/edit-form/component.ts

```diff
@@ -66,12 +66,18 @@ export default class TokenForm extends Component {
 
     @action
     async deleteToken() {
-        try {
-            await this.token!.destroyRecord();
-            this.toast.success(this.intl.t('settings.tokens.deleted'));
-            this.router.transitionTo('settings.tokens');
-        } catch {
-            this.toast.error(this.intl.t('settings.tokens.delete-error'));
+        if (this.token) {
+            try {
+                await this.changeset.validate();
```

Comment:
Why do we need to validate the changeset?

Reply:
I was going to wait on this until we further discussed but seemed helpful to have present for your review. And ok, this is now removed.